core/modules/wildcard_match.h

```diff
@@ -166,10 +168,21 @@ class WildcardMatch final : public Module {
   template <typename T>
   CommandResponse ExtractKeyMask(const T &arg, wm_hkey_t *key, wm_hkey_t *mask);
 
+  template <typename T>
+  CommandResponse Add(const T &arg);
+
+  template <typename T>
+  void GetFields(T &arg);
+
+  template <typename rule_t, typename T, typename Func>
+  void GetRules(T &, Func put_rule);
```

Comment:
In the .cc file the second argument is named `put_field`.

Reply:
Left over from evolution.  If we're really taking out the old get-rules command, can de-templatize and just inline the whole thing anyway.